front/app/components/Form/Components/Layouts/CLSurveyPageLayout.tsx

```diff
@@ -199,79 +250,155 @@ const CLSurveyPageLayout = memo(
-                        >
-                          {page.options.title}
-                        </Title>
-                      )}
-                      {page.options.description && (
-                        <Box mb="48px">
-                          <QuillEditedContent
-                            fontWeight={400}
-                            textColor={theme.colors.tenantText}
-                          >
-                            <div
-                              dangerouslySetInnerHTML={{
-                                __html: page.options.description,
-                              }}
-                            />
-                          </QuillEditedContent>
+        <Box
+          id="container"
+          display="flex"
+          flexDirection={isMobileOrSmaller ? 'column' : 'row'}
+          height="100%"
+          w="100%"
+        >
+          {isMapPage && (
+            <Box
+              id="survey_page_map"
+              w={isMobileOrSmaller ? '100%' : '60%'}
+              minWidth="60%"
+              h="100%"
+              ref={draggableDivRef}
+            >
+              <EsriMap
+                layers={mapLayers}
+                initialData={{
+                  zoom: Number(mapConfig?.data.attributes.zoom_level),
+                  center: mapConfig?.data.attributes.center_geojson,
+                  showLegend: true,
+                  showLayerVisibilityControl: true,
+                }}
+                webMapId={mapConfig?.data.attributes.esri_web_map_id}
+                height="100%"
+              />
+            </Box>
+          )}
+
+          <Box flex={'1 1 auto'} overflowY="auto" h="100%">
+            {isMapPage && isMobileOrSmaller && (
+              <Box
+                aria-hidden={true}
+                height="30px"
+                py="20px"
+                ref={dragDividerRef}
+                position="absolute"
+                background="white"
+                w="100%"
+                zIndex="1000"
+              >
+                <Box
+                  mx="auto"
+                  w="40px"
+                  h="4px"
+                  bgColor="lightgrey"
+                  borderRadius="10px"
+                />
+              </Box>
+            )}
+            <Box
+              display="flex"
+              flexDirection="column"
+              height="100%"
+              mt={isMapPage && isMobileOrSmaller ? '20px' : undefined}
+            >
+              <Box h="100%" display="flex" ref={pagesRef}>
+                {uiPages.map((page, index) => {
+                  const pageElements = extractElementsByOtherOptionLogic(
+                    page,
+                    data
+                  );
+                  return (
+                    currentStep === index && (
+                      <Box key={index} p="24px" w="100%">
+                        <Box display="flex" flexDirection="column">
+                          {allowAnonymousPosting && (
```

Comment:
Moving the Anonymous Posting warning into this component seems to get rid of a lot of the strange styling/scroll issues as well actually :+1: We don't have to do any strange math with the margins/padding anymore.